index.html

```diff
@@ -46,7 +45,153 @@ <h1>I’m&nbsp;Ramoun Glad&nbsp;to&nbsp;see you!</h1>
       </ul>
     </div>
   </section>
-  <!-- intro/end -->
+
+  <!-- works -->
+  <section class="works flex flex-h-center">
+    <div class="wrapper">
+      <article class="card flex bg">
```

Comment:
- [ ] KIndly add some margins for the project section.
- `The section has the margins (top, bottom, left, right) defined in the design`
![Captura de pantalla_2021-10-30_20-55-47](https://user-images.githubusercontent.com/53111464/139563626-58116b09-5cfc-47ca-863c-1c2012ea0863.png)